lib/shared/addon/components/cluster-driver/driver-okecapi/template.hbs

```diff
@@ -336,6 +395,146 @@
       {{/if}}
     {{/if}}
     {{#if (gte step 3)}}
+      {{#accordion-list-item title=(t "clusterNew.ociocne.advanced.title")
+                             detail=(t "clusterNew.ociocne.advanced.detail")
+                             expandAll=expandAll
+                             expand=(action expandFn)
+                             expandOnInit=true}}
+        {{#accordion-list-item title=(t "clusterNew.ociocne.addons.yaml.title")
+                               detail=(t "clusterNew.ociocne.addons.yaml.detail")
+                               expandAll=expandAll
+                               expand=(action expandFn)
+                               expandOnInit=false}}
+          <div class="row">
+            <button
+              class="btn btn-sm bg-primary inline-block mb-5"
+              style="margin-top: -5px;" type="button" {{action "addYAML"}}
+            >
+              {{t "clusterNew.ociocne.addons.yaml.addNew"}}
+            </button>
+          </div>
+          {{#each yamls as |yaml|}}
+            <div class="row">
+              <div class="col span-4">
+                {{input-yaml
+                  showUpload=true
+                  showDownload=true
+                  canChangeName=false
+                  value=yaml.body
+                  autoResize=true
+                  loading=loading
+                }}
+              </div>
+            </div>
+
+            <button
+              class="btn btn-sm bg-primary inline-block mb-5"
+              style="margin-top: -5px;" type="button" {{action "deleteYAML" yaml}}
+            >
+              {{t "clusterNew.ociocne.addons.yaml.delete"}}
+            </button>
+          {{/each}}
+        {{/accordion-list-item}}
+        {{#accordion-list-item title=(t "clusterNew.ociocne.networking.title")
+                               detail=(t "clusterNew.ociocne.networking.detail")
+                               expandAll=expandAll
+                               expand=(action expandFn)
+                               expandOnInit=false}}
+          {{#if (eq vcnCreationMode "Existing")}}
+            <div class="row">
+              <div class="col span-4 mb-0">
+                <label>
+                  {{t 'clusterNew.ociocne.networking.clusterCidr.title'}}{{field-required}}
+                </label>
+                {{input type="text" name="cluster-cidr" classNames="form-control" value=config.clusterCidr}}
```

Comment:
Do i need to get k8s service CIDR block in a different way than it’s done for ocne or will the value exist in the config?

Reply:
i think this is fine